src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2437,6 +2437,10 @@ bool Matcher::supports_vector_variable_shifts(void) {
   return true;
 }
 
+bool Matcher::supports_unsigned_vector_comparison(int vlen, BasicType bt) {
```

Comment:
`Matcher::supports_vector_comparison_unsigned()` looks a better fit.

Reply:
I renamed the method (see commit).